### DISCUSSION TITLE
Diablo 4 Clone Frames

### DISCUSSION THREAD

**Discussion by Aimeryan** [Lailloken/Exile-UI#255] [Ideas & Feedback]
Thoroughly enjoy using the clone frames in POE to move parts the UI that are somewhat out of the way into a more central position.
Any chance of working on something similar for Diablo 4? Info like combo points, energy, etc could drastically be improved by this.

Cheers 👍

**Comment by Lailloken** [comment]
Sorry, I don't play Diablo. Someone else once asked me about making a standalone, game-independent version of clone-frames. There's a slim chance I'll do that at some point, but not any time soon.
